src/sentry/utils/samples.py

```diff
@@ -126,7 +126,8 @@ def load_data(platform, default=None, sample_name=None, timestamp=None, start_ti
 
         # Verify by checking if the file is within our folder explicitly
         # avoids being able to have a name that invokes traversing directories.
-        json_path = "%s.json" % platform.encode("utf-8")
+        json_path = u"{}.json".format(platform)
```

Comment:
Just noting here that in this specific case, we always pass str, so this is fine.

100% correct would be wrapping platform in a `force_text` for format.  I wonder if we can actually write a lint for this.